app/models/disputed_asset_model.rb

```diff
@@ -3,28 +3,18 @@ class DisputedAssetModel
   include ActiveModel::Attributes
```

Comment:
is disputed_asset.. still the best name for this model as it it only deals with properties now?